roles/test-provision-apb/tasks/main.yml

```diff
@@ -8,9 +8,6 @@
   register: webpage
   retries: 10
   delay: 20
-  failed_when:
```

Comment:
Do we not need a failed condition?

Reply:
I think it was necessary to have it only for `metrics-apb`, because oauth proxy server was returning strange response which was causing the task to fail.